contracts/ethereum-sc/contracts/ERC20Safe.sol

```diff
@@ -164,12 +164,12 @@ contract ERC20Safe {
         - timestamp
         - deposits List of the deposits included in this batch
         @dev This function is to be called by the bridge (which is called by the relayers)
-        It only returns final batches - batches where the block time limit has passed.
+        It only returns final batches - batches where the block time limit has passed and that have settled (minimal risk for reorgs).
     */
     function getNextPendingBatch() public view returns (Batch memory) {
         Batch memory batch = batches[currentPendingBatch];
 
-        if ((batch.lastUpdated + batchSettleLimit) < block.timestamp)
+        if ((batch.lastUpdatedBlockNumber + batchSettleBlockCount) <= block.number)
```

Comment:
Any reason you chose `<=` instead of `<` ?

Reply:
quoting @AdiDance 

> Yes, this is a view function. It doesn't need to mine a new block in order to execute. That's why, when it run, it's enough to be `<=`